1.10.10.dev/buster/Dockerfile

```
@@ -103,17 +103,17 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-ARG VERSION="<1.10.11,>=1.10.10"
+ARG VERSION="1.10.10.*"
 ARG SUBMODULES="async,azure_blob_storage,azure_cosmos,azure_container_instances,celery,crypto,elasticsearch,gcp,kubernetes,mysql,postgres,s3,emr,redis,slack,ssh,statsd,virtualenv"
-ARG AIRFLOW_MODULE="astronomer_certified[${SUBMODULES}]$VERSION"
+ARG AIRFLOW_MODULE="astronomer_certified[${SUBMODULES}]==$VERSION"
 
 # Make pip look at our pip repo too, and force it to install these specific
 # versions when ever it installs a module.
 COPY include/pip.conf /etc/pip.conf
 COPY include/pip-constraints.txt /usr/local/share/astronomer-pip-constraints.txt
 
 # Pip install airflow and astro security manager
-RUN pip install "${AIRFLOW_MODULE}" --pre \
+RUN pip install --no-cache-dir "${AIRFLOW_MODULE}" --pre \
   && pip install "https://github.com/astronomer/astronomer-airflow-scripts/releases/download/v0.0.4/astronomer_airflow_scripts-0.0.4-py3-none-any.whl" \
 	&& pip install --no-cache-dir "astronomer-fab-security-manager~=1.2, >=1.2.2"
```

```suggestion
	&& pip install "astronomer-fab-security-manager~=1.2, >=1.2.2"
```

since we set that globally in via env in the container

---

Updated